packages/components/src/components/TaskRunDetails/TaskRunDetails.test.js

```diff
@@ -62,4 +62,18 @@ describe('TaskRunDetails', () => {
     fireEvent.click(queryByText(/parameters/i));
     expect(queryByText('fake_name')).toBeTruthy();
   });
+
+  it('renders results', () => {
```

Comment:
should we add a falsly assertion on the regular case for the results tab?

Reply:
added a test to cover both missing params and results 👍